pkgs/development/python-modules/genie-partner-sdk/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  hatchling,
+  aiohttp,
+  fetchPypi,
+}:
+
+buildPythonPackage rec {
+  pname = "genie-partner-sdk";
+  version = "1.0.3";
+  pyproject = true;
+
+  disabled = pythonOlder "3.11";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "genie_partner_sdk";
+    hash = "sha256-/Pu0pULepRpAdUfu2SQjK4xCSe3FWWNT9AqZEASoZSE=";
+  };
+
+  nativeBuildInputs = [ hatchling ];
+
+  propagatedBuildInputs = [ aiohttp ];
+
+  # No tests
+  doCheck = false;
+
+  pythonImportsCheck = [ "genie_partner_sdk" ];
+
+  meta = with lib; {
+    description = "An SDK to interact with the AladdinConnect (or OHD) partner API";
+    homepage = "https://github.com/Genie-Garage/aladdin-python-sdk";
+    license = licenses.unfreeRedistributable;
```

Comment:
I believe this is the nixpkgs equivalent for the license: https://github.com/Genie-Garage/aladdin-python-sdk/blob/main/LICENSE

Reply:
Wow, that is super annoying. Looks like it originates here: https://github.com/kolappannathan/closed-source-license.

Doesn't "All rights reserved" imply that we cannot redistribute it?

Reply:
Okay, I'm going to open an issue upstream in Home Assistant to clarify, because the Home Assistant container image also redistributes this package. See `usr/local/lib/python3.12/site-packages/genie_partner_sdk-1.0.2.dist-info/licenses/LICENSE` in `ghcr.io/home-assistant/home-assistant`

Reply:
Opened: https://github.com/home-assistant/core/issues/119697

Reply:
@mweinelt I've updated the license to `unfree`. It's the most restrictive version of that license, and should allow us to continue with the merge. I'll update it to `unfreeRedistributable` if I get clarification upstream.